src/System.Net.Http.WinHttpHandler/src/System/Net/Http/WinHttpResponseStream.cs

```diff
@@ -85,6 +67,9 @@ public override Task FlushAsync(CancellationToken cancellationToken)
                 Task.CompletedTask;
         }
 
+        public override void CopyTo(Stream destination, int bufferSize) =>
+            CopyToAsync(destination, bufferSize).GetAwaiter().GetResult();
```

Comment:
This file needs to compile for desktop as well, where this method isn't virtual.  And I don't think the value added here is significant enough to warrant splitting this out into a separate file... I'd just revert this file.